### ISSUE TITLE
Fatal error in php beautifier using AllmanStyleBraces

### ISSUE THREAD

**Issue by SylwesterZarebski** [akalongman/sublimetext-codeformatter#278]
This piece of code crashes php beautifier when using AllmanStyleBraces:
```
$test->test('test', function ($this) { return $this->test; });
```
Crash message:
```
---------------------------
Sublime Text
---------------------------
CodeFormatter

Format error:
Fatal error: Uncaught TypeError: substr_count() expects parameter 3 to be integer, boolean given in phar://C:/Users/Sylwek/AppData/Roaming/Sublime Text 3/Packages/CodeFormatter/codeformatter/lib/phpbeautifier/fmt-php7.phar/fmt.stub.php:6058
Stack trace:
#0 phar://C:/Users/Sylwek/AppData/Roaming/Sublime Text 3/Packages/CodeFormatter/codeformatter/lib/phpbeautifier/fmt-php7.phar/fmt.stub.php(6058): substr_count(' ', '\t', false)
#1 phar://C:/Users/Sylwek/AppData/Roaming/Sublime Text 3/Packages/CodeFormatter/codeformatter/lib/phpbeautifier/fmt-php7.phar/fmt.stub.php(1656): AllmanStyleBraces->format('<?php\n$test->te...')
#2 phar://C:/Users/Sylwek/AppData/Roaming/Sublime Text 3/Packages/CodeFormatter/codeformatter/lib/phpbeautifier/fmt-php7.phar/fmt.stub.php(12433): BaseCodeFormatter->formatCode('<?php\n$test->te...')
#3 C:\Users\Sylwek\AppData\Roaming\Sublime Text 3\Packages\CodeFormatter\codeformatter\lib\phpbeautifier\fmt-php7.phar(10): include('phar://C:/Users...')
#4 {main}
  thrown in phar://C:/Users/Sylwek/AppData/Roaming/Sublime Text 3/Packages/CodeFormatter/codeformatter/lib/phpbeautifier/fmt-php7.phar/fmt.stub.php on line 6058
```

Without "AllmanStyleBraces" option it works, but i can not use standard coding style.

**Comment by sixhead** [comment]
Me too. Error after upgrade to v3.5.0

```
<?php
echo 1;
```

```
Format error:
PHP Fatal error:  A function with return type must return a value in phar://C:/Users/sixhead/AppData/Roaming/Sublime Text 3/Packages/CodeFormatter/codeformatter/lib/phpbeautifier/fmt-php7.phar/fmt.stub.php on line 1354
PHP Stack trace:
PHP   1. {main}() C:\Users\sixhead\AppData\Roaming\Sublime Text 3\Packages\CodeFormatter\codeformatter\lib\phpbeautifier\fmt-php7.phar:0

Fatal error: A function with return type must return a value in phar://C:/Users/sixhead/AppData/Roaming/Sublime Text 3/Packages/CodeFormatter/codeformatter/lib/phpbeautifier/fmt-php7.phar/fmt.stub.php on line 1354

Call Stack:
    0.0023     423816   1. {main}() C:\Users\sixhead\AppData\Roaming\Sublime Text 3\Packages\CodeFormatter\codeformatter\lib\phpbeautifier\fmt-php7.phar:0
```

**Comment by akalongman** [comment]
Will fix ASAP

**Comment by momoca** [comment]
Me too. 
fix use php7.0.4.
```
{
    "codeformatter_php_options": {
        "syntaxes": "php", // Syntax names which must process PHP formatter
        "php_path": "D:/php/php7.0.4/php.exe", 
        ...
```

**Comment by momoca** [comment]
May be the original should be used `fmt-php55.phar` but using the `fmt-php7.phar` caused by the error.

**Comment by SylwesterZarebski** [comment]
Looks like bug is fixed with 3.5.1 version. Thanks.
